### PR TITLE
rename: proxyAdminOwner -> superchainProxyAdminOwner

### PR DESCRIPTION
**Pull request by blmalone**
rename: proxyAdminOwner -> superchainProxyAdminOwner

fix: renames in tests.

### CODE REVIEW

**Comment by blmalone** [comment]
* **#12121** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/12121?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#12107** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/12107?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#12106** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/12106?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#12091** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/12091?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#12092** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/12092?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @blmalone and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: op-chain-ops/deployer/pipeline/superchain.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ func DeploySuperchain(ctx context.Context, env *Env, artifactsFS foundry.StatDir
 				dso, err = opcm.DeploySuperchain(
 					host,
 					opcm.DeploySuperchainInput{
-						ProxyAdminOwner:            intent.SuperchainRoles.ProxyAdminOwner,
+						SuperchainProxyAdminOwner:  intent.SuperchainRoles.ProxyAdminOwner,
```
Comment (mds1) [inline op-chain-ops/deployer/pipeline/superchain.go:43]: Should we change this in the intent also?
```suggestion
						SuperchainProxyAdminOwner:  intent.SuperchainRoles.SuperchainProxyAdminOwner,
```

File: op-chain-ops/interopgen/deploy.go [hunk 2]
```diff
@@ -149,7 +149,7 @@ func deploySuperchainToL1(l1Host *script.Host, superCfg *SuperchainConfig) (*Sup
 	l1Host.SetTxOrigin(superCfg.Deployer)
 
 	superDeployment, err := opcm.DeploySuperchain(l1Host, opcm.DeploySuperchainInput{
-		ProxyAdminOwner:            superCfg.ProxyAdminOwner,
+		SuperchainProxyAdminOwner:  superCfg.ProxyAdminOwner,
```
Comment (mds1) [inline op-chain-ops/interopgen/deploy.go:152]: similar question
```suggestion
		SuperchainProxyAdminOwner:  superCfg.SuperchainProxyAdminOwner,
```

File: op-chain-ops/deployer/pipeline/superchain.go (hunk 1 above)
Comment (blmalone) [reply op-chain-ops/deployer/pipeline/superchain.go:43]: I wasn't sure so I didn't. I can change though if we want.

Comment (blmalone) [reply op-chain-ops/deployer/pipeline/superchain.go:43]: My reasoning was that the `DeploySuperchainInput` was descriptive enough such that the intent didn't need to be. We don't lose anything adding it to the intent so I'll make that change.

Comment (blmalone) [reply op-chain-ops/deployer/pipeline/superchain.go:43]: https://github.com/ethereum-optimism/optimism/pull/12106/commits/19257da7b70b2bee848f41f331e4bd23ccf033bc

File: op-chain-ops/interopgen/deploy.go (hunk 2 above)
Comment (blmalone) [reply op-chain-ops/interopgen/deploy.go:152]: https://github.com/ethereum-optimism/optimism/pull/12106/commits/19257da7b70b2bee848f41f331e4bd23ccf033bc

File: op-chain-ops/deployer/pipeline/superchain.go (hunk 1 above)
Comment (mds1) [reply op-chain-ops/deployer/pipeline/superchain.go:43]: Thanks. I don't feel strongly, so will defer to you and @mslipper here on preferences

**Review by mds1 — APPROVED** [review]

File: op-chain-ops/deployer/pipeline/superchain.go (hunk 1 above)
Comment (mslipper) [reply op-chain-ops/deployer/pipeline/superchain.go:43]: i leave as-is in the intent - seems weird to prefix everything in a data structure called "superchain roles" with "superchain"

**Review by mslipper — APPROVED** [review]

File: op-chain-ops/deployer/pipeline/superchain.go (hunk 1 above)
Comment (blmalone) [reply op-chain-ops/deployer/pipeline/superchain.go:43]: https://github.com/ethereum-optimism/optimism/pull/12106/commits/650a5e44473ea1482525be015218094df580cb5e

**Pull request merged**